org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/refactoring2/ChangeConverter.java

```diff
@@ -130,7 +130,7 @@ protected void _handleReplacements(IEmfResourceChange change) {
 				change.getResource().save(outputStream, null);
 				byte[] newContent = outputStream.toByteArray();
 				String lastSegment = change.getOldURI().lastSegment();
-				ReplaceFileContentChange ltkChange = new ReplaceFileContentChange(lastSegment, file, newContent);
+				ReplaceFileContentChange ltkChange = ReplaceFileContentChange.createWithSkippingUndoVerify(lastSegment, file, newContent);
```

Comment:
wonder if this should go into a methid so that it can be more easily cusomized.
@szarnekow @LorenzoBettini wdyt?

Reply:
Yes. In that case some javadoc should explain it.

Reply:
You mean an overridable method?
````
protected Change createLtkChange(name, file, content){ ....
````